config/server.properties

```diff
@@ -16,3 +16,4 @@ host.name=localhost
 clustermap.cluster.name=Ambry_Dev
 clustermap.datacenter.name=Datacenter
 clustermap.host.name=localhost
+clustermap.port=6667
```

Comment:
Why is this newly added?

Reply:
ah, I just added it here to fix issue #1357  (probably should be in a separate PR but let's get it fixed together with this PR merged)

Reply:
Feel free to ignore this, not related to this PR.

Reply:
It's OK. I also need this change. :)

Reply:
Thanks for fixing this.